packages/manager/apps/pci-private-registry/src/api/data/registry.ts

```diff
@@ -65,5 +65,24 @@ export const deleteRegistry = async (projectId: string, registryId: string) => {
   const { data } = await v6.delete(
     `/cloud/project/${projectId}/containerRegistry/${registryId}`,
   );
+
+  return data;
+};
+
+export type TRegistryCredentials = {
+  email: string;
+  id: string;
+  password: string;
+  user: string;
+};
+
+export const getRegistryCredentials = async (
+  projectId: string,
+  registryId: string,
+): Promise<TRegistryCredentials> => {
+  const { data } = await v6.post(
```

Comment:
a get may be?

Reply:
no it's a post, i've tried the get but it's dosen't work